requirements/dev.txt

```diff
@@ -28,3 +29,9 @@ pytest-django==2.8.0 \
     --hash=sha256:d145ac9dc7a557a719ab79770be0941004e1e038e137c34591919d9df2a790b1
 pytest-pythonpath==0.7 \
     --hash=sha256:036bc2b62b5d3a991e45c389748ccfb61e88eb5b14b27cab4b5c1fbcd4ffd191
+funcsigs==1.0.1 \
+    --hash=sha256:2edd42db946babc214077be3626e1c496561daeb6e752e482d8d733a0d578f01 \
+    --hash=sha256:0726847f1463526794496423f6500cc2ed751361b6c025982ab18bc6c5af35b1
+pbr==1.9.1 \
```

Comment:
why are there new requirements as part of this PR? I don't see where they are used.